lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -423,19 +423,24 @@ public function security_validation( $old_options, $input ) {
 					__( 'No database connections found for this application. ', 'wp-auth0' ) .
 					$this->get_dashboard_link( 'connections/database', __( 'See all database connections', 'wp-auth0' ) )
 				);
+				$input['password_policy'] = $old_options['password_policy'];
+				return $input;
```

Comment:
Exit here to skip anything else that might process (`foreach` will throw an error if it's given anything other than an array)